gitrevise/todo.py

```diff
@@ -141,21 +131,21 @@ def autosquash_todos(todos: List[Step]) -> List[Step]:
                 needle
             ) or target.commit.oid.hex().startswith(needle):
                 found = idx
+                number_of_transitive_fixup_commits = (
+                    count_fixup_commits(fixups, target.commit.oid) - 1
+                )
```

Comment:
I'd skip the `-1` here, as you're adding one to `number_of_transitive_fixup_commits` below when inserting.

I think we can probably also shorten the variable's name to something more like `transitive_fixup_count`

Reply:
Ok makes sense. 
Now `count_fixup_commits` is the number of fixup commits to the target, plus one for the target commit itself.